.eslintrc.cjs

```diff
@@ -1,19 +1,23 @@
-{
+/* eslint-env node */
+
+module.exports = {
   "env": {
     "browser": true,
-    "es6": true
+    "es2021": true
   },
-  "extends": "eslint:recommended",
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": 2018,
+    "ecmaVersion": 12,
```

Comment:
it's unlikely this will bite you, but I just learning a lesson about this yesterday in fact.

what you are telling eslint here is that your source code works with any fancy new [ESNEXT](https://node.green/#ESNEXT) syntax features and that it shouldn't complain about them during linting or in the IDE..... but.... you state that this project RUNS on `Node 14` (due to the advice of some guy on the Internet... 😅), which does not support some of the language features you might try to use and think are okay because your linter says so.

confusing? yea. I dont know how TS factors into that too, its another layer that maybe mitigates this but I had trouble finding a resource like node.green is for TS.

The safest bet would be to downgrade `esnext` references back to what you KNOW Node 14 supports, which is [ES2019](https://node.green/#ES2019)